README.md

```diff
@@ -1,2 +1,100 @@
-# hello-microversee
```

Comment:
Hey, @Eternalgratis I noticed the template you are using is not the one provided, I think it would be better to use the [provided template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) as this will help you document your project better in a more industry accepted manner 👩‍💻